client/settings/fraud-protection/components/fp-modal/index.js

```diff
@@ -0,0 +1,238 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { Button, Modal } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import ExceedsDollarAmountRule from '../exceeds-dollar-amount-rule';
+import ProtectionLevelModalNotice from '../protection-level-modal-notice';
+
+export const HighFraudProtectionModal = ( {
+	level,
+	isHighModalOpen,
+	setHighModalOpen,
+	storeCurrency,
+} ) => {
+	return (
+		<>
+			{ isHighModalOpen && (
+				<Modal
+					title={ __( 'High filter level', 'woocommerce-payments' ) }
+					isDismissible={ true }
+					shouldCloseOnClickOutside={ true }
+					shouldCloseOnEsc={ true }
+					onRequestClose={ () => setHighModalOpen( false ) }
+					className="fraud-protection-level-modal"
+				>
+					<div className="components-modal__body--fraud-protection">
+						<ProtectionLevelModalNotice level={ level } />
+						<p>
+							{ __(
+								'Payments will be ',
+								'woocommerce-payments'
+							) }
+							<span className="component-modal__text--blocked">
+								{ __( 'blocked ', 'woocommerce-payments' ) }
+							</span>
+							{ __( 'if: ', 'woocommerce-payments' ) }
```

Comment:
Another instance where we may be able to use `interpolateComponents` to generate one string. I am unsure if it's possible to pass a class name into the html, though.

Reply:
I don't believe it is possible to pass a class name into the html, but I'll research further to find our for sure.

Reply:
I researched many of the numerous uses of `interpolateComponents` in the codebase and you can definitely pass a class name within the components property and do all sorts of neat stuff in there. I fixed this in [385c894](https://github.com/Automattic/woocommerce-payments/pull/5450/commits/385c894ff631f59b7012878f69e677a32337de59).